README.md

```diff
@@ -1,2 +1,25 @@
 # cowpoke
-A service for wrangling DNS 
+
+A service for wrangling DNS
+
+## Developing
+
+### Installing
+
+1. Install the requirements:
+
+    1. [docker](https://docs.docker.com/get-docker/)
+    1. [docker-compose](https://docs.docker.com/compose/install/)
+    1. [Python 3.9](https://www.python.org/downloads/)
+    1. [poetry](https://python-poetry.org/docs/#installation)
+
+1. run `poetry install` to create a virtualenv and install python dependencies
```

Comment:
For this to work I ended up needing to install postgres.  ```brew install postgresql``` was enough.  Error was basically that psycopg2 can't install without the pg_config binary.

Reply:
Oh yeah, good catch! I must've already had those installed.

Updated the readme.